blosc/blosc_extension.c

```diff
@@ -189,15 +209,38 @@ compress_helper(void * input, size_t nbytes, size_t typesize,
   }
 
   /* Compress */
-  cbytes = blosc_compress(clevel, shuffle, typesize, nbytes,
-			  input, PyBytes_AS_STRING(output),
+  // RAM: I don't think this macro requires the Python interpreter but let's leave it outside
```

Comment:
You could remove the `RAM` comments and persist your thoughts. We can always use `git blame` to figure out who wrote the comments. This is valid for all your comments.